pkg/util/env/env.go

```diff
@@ -68,6 +68,10 @@ func GetDefaultMeshNamespace() string {
 	return Getenv("MESHNAMESPACE", "istio-system")
 }
 
+func GetDefaultSMCPProfile() string {
+	return Getenv("SMCPPROFILE", "full")
+}
+
```

Comment:
This must be configurable at least for running a single test. When I test cert-manager, I don't care about tracing or dashboards, so I only waste time on installation those components.

Reply:
No, because this depends on each test. If a test requires only a minimal SMCP, then it should install it. You can't make the profile configurable across all tests, because you'll break the tests that require a specific profile.

Reply:
I'm not saying that you must run the test against the full profile. I'm just saying you must not add a dangerous env var like `SMCPPROFILE`, because somebody will eventually use it to achieve something that it wasn't intended for. 

Why don't you simply create a minimal SMCP in your test?

Reply:
Because full SMCP is always deployed before my test.

Reply:
Then the test is in the wrong package.

Reply:
So how to run only TestCertManager? I use command `make test TestCertManager`.

Reply:
Ok, I will change main test suite in the security package.